src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+import duke.command.DukeCommands;
+import duke.util.*;
+
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Scanner;
+
+public class Duke {
+    /**
+     * Simple string array to store inputs
+     **/
+    private final DukeTaskList list;
+
+    private final DukeDB database;
+
+    private final DukeConfig config;
+
+    private final Ui ui;
+    /**
+     * Basic constructor to initialise the list
+     **/
+    public Duke(DukeDB database, DukeConfig config) {
+        this.ui = new Ui();
+        this.database = database;
+        this.list = new DukeTaskList(database.load().orElse(new ArrayList<>()));
+        this.config = config;
+
+    }
+
+
+    public Duke(DukeDB database) {
+        this.ui = new Ui();
+        this.database = database;
+        this.list = new DukeTaskList(database.load().orElse(new ArrayList<>()));
+        this.config = new DukeConfig();
+    }
+
+
+    /**
+     * Simple formatting tool to be used when printing commands
```

Comment:
Could consider phrasing as an action instead? e.g. Prints ...

Reply:
I notice the same issue as well on other public methods.